LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
+
+Copyright (c) 2024, ¯\_(ツ)_/¯
```

Comment:
Note to self: need to decide who the copyright holder is and insert here instead of shrug emoji, even if it is just "book authors"

Reply:
+1 for ’authors’.